### DISCUSSION TITLE
Can we install musics on device?

### DISCUSSION THREAD

**Discussion by DavutHaxor** [ciderapp/Apple-Music-Electron#218] [Q&A]
Is there any way to do it?

**Comment by coredev-uk** [comment]
If you're talking about mobile devices, you can find the Apple Music app on the Google Play store, or it would be pre-installed on Apple devices. If you're referring to this app, check releases by clicking [here](https://github.com/Apple-Music-Electron/Apple-Music-Electron/releases/latest).
